src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build.py

```diff
@@ -263,13 +265,25 @@ def acr_queue(cmd,
         result = LongRunningOperation(cmd.cli_ctx)(client_registries.queue_build(
             build_request=build_request, resource_group_name=resource_group_name, registry_name=registry_name))
 
-        print("Queued a build with build-id: {}.".format(result.build_id))
```

Comment:
Should leave this print statement in, otherwise there's no output if `no_logs == False`

Reply:
Yes I am not sure what would be the proper behavior if no_logs == False. I will look into that and talk to Steve later.

Reply:
Sorry I mean if no_logs is not False

Reply:
It has already resolved in the version I committed last night.

Reply:
It has already been resolved. The "Build context..." would still be printed even if the no-logs flag is set to true.